src/screens/Library.jsx

```diff
@@ -0,0 +1,101 @@
+import React, {useState, useEffect} from "react";
+import likesStyles from "./Likes.module.css";
+import profileStyles from "./Profile.module.css";
+import Card from "../components/Card";
+import { API } from "../static/Services";
+import Loader from '../components/Loader';
+import buttonStyles from "../components/Button.module.css";
+
+const defaultState = {
+    labs: [],
+    candidates: [],
+    loading: <Loader />,
+    isCompanyMode: true
+}
+
+const style = {
```

Comment:
I think you are not using this anywhere.

Reply:
I'm using it in the companyCandidates const as style for the divs

Reply:
I saw where you use it, however, its probably not the best way of doing things, you should add these things to the .css file

Reply:
You are right